src/components/LibrariesPage.tsx

```diff
@@ -259,8 +262,14 @@ export class LibrariesPage extends React.Component<
       showAll: false,
       searchTerm: data.get("name") as string,
       filters: this.state.filters,
+      receivedSearchResults: false,
     });
-    await this.props.search(data);
+    try {
+      await this.props.search(data);
+    } catch (e) {
+      console.error(`No results found for '${this.state.searchTerm}'.`);
```

Comment:
Not receiving results from the API is technically not an error. It's possible a user search for a library that doesn't exist and the API returns 0 results. Instead of logging this as an error, it should be a warning `console.warn`.

But, the question that comes to mind is how useful this would be in the console since this message is meant for users. Maybe log the `e` error instead if that's more helpful to a dev.